iml-gui/crate/src/components/lnet_status.rs

```diff
@@ -0,0 +1,26 @@
+use crate::{components::font_awesome, generated::css_classes::C, MergeAttrs};
+use iml_wire_types::db::LnetConfigurationRecord;
+use seed::{prelude::*, *};
+
+fn network<T>(color: impl Into<Option<&'static str>>) -> Node<T> {
+    if let Some(color) = color.into() {
+        font_awesome(
+            class![C.w_4, C.h_4, C.inline, C.mr_1].merge_attrs(class![color]),
```

Comment:
I have seen in some PR, that you created a trait for merging attrs, no?

Reply:
pushed.